app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+class CommentsController < ApplicationController
+  #index
+  def index
+  @post = Post.find(params[:post_id])
+  @comment = Comment.all
```

Comment:
Thanks,

  def index
 @post = Post.find(params[:post_id])
  @comment = Comment.all
is just there if you wanna see all the comments on all the posts together, the show path is actually the one which will take you to that posts comment. But ya now that I think about it I didn't need to add that